### DISCUSSION TITLE
android端 ，请问下怎么做到离屏渲染，然后获取到纹理id ?

### DISCUSSION THREAD

**Discussion by gzzhangbl** [Tencent/tgfx#194] [Q&A] [answered]
android端 ，请问下怎么做到离屏渲染，然后获取到纹理id ?谢谢

**Comment by domchen** [comment, marked as answer]
通过宽高创建Surface，然后渲染完flush后执行getBackendTexture()。或者你直接自己创建纹理id，包装成BackendTexture用来创建Surface，每个平台都一样。
